### PR TITLE
Add database migrations to repository

### PR DESCRIPTION
**Pull request by jennchenn**
## Notion ticket link

[Add database migrations to repository](https://www.notion.so/uwblueprintexecs/Add-database-migrations-to-repository-9dbf580ccd994b90aa613e97a26fc7f2)

## Implementation description
* Generate initial database migration using Flask-Migrate

## Steps to test
1. Add the seed_db.sh `AUTH_ID`s to your root `.env` file
2. Rebuild containers: `docker-compose down --volumes` and `docker-compose up --build`
3. Run `docker exec -it planet-read_py-backend_1 /bin/bash -c "flask db upgrade"`
4. Verify that the database tables are generated by checking the database itself
5. Run `docker exec -it planet-read_py-backend_1 /bin/bash -c "flask db downgrade"` and verify the tables are dropped: `docker exec -it planet-read_db_1 mysql -u root -proot -e "USE planet-read; SHOW TABLES;"` 

## What should reviewers focus on?
* `backend/python/migrations/versions/3f6fff20884c_initial_migration.py` (most of the other files in `backend/python/migrations/` were auto-generated/modified based on Starter Code)
~~* Is there a better place to put the seeded data than in the initial migration itself? This seemed to be the pattern that turned up the most when researching. A reference to the create table object is used in the `op.bulk_insert()` command, but an alternate command can be used if we want to separate it out (i.e. `op.execute()`. However, I believe alembic revisions are linear, so if we create a separate migration specifically for test data and then decide to remove it once we release to production, we will need to make sure we update the revision heads of the migrations to ensure the `flask db upgrade` command runs.~~
* The `AUTH_ID_[1-7]`s that were used for the seed_db.sh would need to be added to the vault

## Checklist
- [x] My PR name is descriptive and in imperative tense
- [x] My commit messages are descriptive and in imperative tense. My commits are atomic and trivial commits are squashed or fixup'd into non-trivial commits
- [x] For backend changes, I have run the appropriate linters:  `docker exec -it <backend-container-id> /bin/bash -c "black . && isort --profile black ."`
- [x] I have requested a review from the PL, as well as other devs who have background knowledge on this PR or who will be building on top of this PR

### CODE REVIEW

File: backend/python/Dockerfile [hunk 1]
```diff
@@ -6,7 +6,6 @@ COPY requirements.txt ./
 RUN pip install -r requirements.txt
 
 COPY . ./app
-VOLUME file_uploads:/var/lib/file_uploads
```
Comment (jennchenn) [inline backend/python/Dockerfile:9]: i think this double declaration in here and the compose file might have caused the file volume OCI runtime error. however i got the error pretty inconsistently before making this change so not sure if it's just coincidence that i haven't seen it again. removing it doesn't seem to cause any issues with running the containers though

File: backend/python/migrations/versions/3f6fff20884c_initial_migration.py [hunk 2]
```diff
@@ -0,0 +1,523 @@
...
+    op.create_index(
+        op.f("ix_comments_story_translation_content_id"),
+        "comments",
+        ["story_translation_content_id"],
+        unique=False,
+    )
+    ### end Alembic commands ###
+
+    # seed database
+    op.bulk_insert(
```
Comment (gaoxk) [inline backend/python/migrations/versions/3f6fff20884c_initial_migration.py:171]: image
woops, didnt realize seeding would be included in this pr, closing #81. One modification I'll ask for: can you copy the descriptions and youtube links for stories from #81?

File: README.md [hunk 3]
```diff
@@ -63,6 +63,25 @@ Frontend has on-save linting. To lint the backend:
...
+```
+docker exec -it planet-read_py-backend_1 /bin/bash -c "flask db downgrade"
+``` 
+
+
+A new migration will need to be generated when database changes are made. Import any new tables into [backend/python/app/models/\_\_init__.py](backend/python/app/models/__init__.py) and run: 
+```
+docker exec -it planet-read_py-backend_1 /bin/bash -c "flask db migrate -m '<description of your migration>'"
+```
+Ensure that a new revision file is created in the directory [backend/python/migrations/versions](backend/python/migrations/versions).
```
Comment (gaoxk) [inline README.md:81]: nit: I'd add that generally these auto generated files should be left untouched (initial db seeding is an exception)

**Review by gaoxk — CHANGES_REQUESTED** [review]
> if we create a separate migration specifically for test data and then decide to remove it once we release to production, we will need to make sure we update the revision heads of the migrations to ensure the flask db upgrade command runs.

Can you describe how we would go about swapping initial seed/test data migrations (would it actually just be a alembic script?) for production vs development environments? 
To distinguish test data vs seed data: test data is what we currently have going on - it's fake data that helps us test and develop. seed data is data that must be seeded to make the application useable (eg, initial admin accounts). inserting test data should not happen on prod, but inserting seed data should.
A very hacky way to work with the separate migrations is to make sure the test data migration (current seed_db results) dont get copied to the client's server. Curious what your thoughts are.

**Comment by gaoxk** [comment]
((also, tested it and it worked!))

File: backend/python/migrations/versions/3f6fff20884c_initial_migration.py (hunk 2 above)
Comment (jennchenn) [reply backend/python/migrations/versions/3f6fff20884c_initial_migration.py]: Removed seeding; adding to this PR instead: https://github.com/uwblueprint/planet-read/pull/83

File: README.md [hunk 4]
```diff
@@ -63,6 +63,25 @@ Frontend has on-save linting. To lint the backend:
...
+```
+docker exec -it planet-read_py-backend_1 /bin/bash -c "flask db downgrade"
+``` 
+
+
+A new migration will need to be generated when database changes are made. Import any new tables into [backend/python/app/models/\_\_init__.py](backend/python/app/models/__init__.py) and run: 
+```
+docker exec -it planet-read_py-backend_1 /bin/bash -c "flask db migrate -m '<description of your migration>'"
+```
+Ensure that a new revision file is created in the directory [backend/python/migrations/versions](backend/python/migrations/versions). Do _not_ change the alembic revision/identifiers. Generally these auto-generated revision files will encompass all schema changes, and thus do not need to be modified!
```
Comment (gaoxk) [inline README.md:81]: nit: bold **Do not** rather than italicize just not

**Review by gaoxk — APPROVED** [review]
noooooooooooooooooice!

**Pull request merged**